kin/client.py

```diff
@@ -95,45 +85,53 @@ def get_config(self):
+    def get_minimum_fee(self):
+        """
+        Get the current minimum fee acceptable for a tx
+        :return: The minimum fee
+        :type: int
+        """
+        params = {'order': 'desc',
+                  'limit': 1}
+        return self.horizon.ledgers(params=params)['_embedded']['records'][0]['base_fee_in_stroops']
+
+    def get_account_balance(self, address):
         """
-        Get the KIN and XLM balance of a given account
+        Get the KIN balance of a given account
         :param str address: the public address of the account to query
-        :return: a dictionary containing the balances
-        :rtype: dict
+        :return: the balance of the account
+        :rtype: float
 
-        :raises: ValueError: if the provided address has the wrong format.
+        :raises: StellarAddressInvalidError: if the provided address has the wrong format.
         :raises: :class:`KinErrors.AccountNotFoundError`: if the account does not exist.
         """
+
+        if not is_valid_address(address):
+            raise KinErrors.StellarAddressInvalidError('invalid address: {}'.format(address))
         account_data = self.get_account_data(address)
-        balances = {}
         for balance in account_data.balances:
-            if balance.asset_code == self.kin_asset.code and \
-                            balance.asset_issuer == self.kin_asset.issuer:
-                balances['KIN'] = balance.balance
-            elif balance.asset_type == 'native':
-                balances['XLM'] = balance.balance
+            # accounts will always have native asset
+            if balance.asset_type == 'native':
+                return balance.balance
 
-        return balances
-
-    def get_account_status(self, address):
+    def does_account_exists(self, address):
         """
-        Get a given account status
-        :param str address: The public address of the account to query.
-        :return: One the possible account statuses.
-        :rtype: :enum:`kin.AccountStatus`
+        Find out if a given account exists on the blockchain
+        :param str address: The kin account to query about
+        :return: does the account exists on the blockchain
+        :rtype boolean
+
+        :raises: :class:`KinErrors.StellarAddressInvalidError`: if the address is not valid.
         """
-        try:
-            balances = self.get_account_balances(address)
-        except KinErrors.AccountNotFoundError:
-            return AccountStatus.NOT_CREATED
 
-        try:
-            balances['KIN']
-        except KeyError:
-            return AccountStatus.NOT_ACTIVATED
+        if not is_valid_address(address):
+            raise KinErrors.StellarAddressInvalidError('invalid address: {}'.format(address))
 
-        return AccountStatus.ACTIVATED
+        try:
+            self.get_account_balance(address)
+            return True
+        except KinErrors.AccountNotFoundError:
+            return False
```

Comment:
if another exception will be raised this function will return `None`. Do this instead:
```python
try:
    ...
    return True
except MyError:
    return False

return False  # catch all - just make sure this flow makes sense in this context
```
also, good oppurtunity to read on try/except/else/finally: https://docs.python.org/3/tutorial/errors.html#handling-exceptions

Reply:
If any other error is raised, I don't want to return False, I want it to just be raised.